package.json

```diff
@@ -72,6 +72,7 @@
     "passport": "^0.5.0",
     "protobufjs": "~6.11.2",
     "rc": "1.2.8",
+    "re2": "^1.17.1",
```

Comment:
Why adding re2 package? Is it because standard regexp are vulnerable?

It remind me something with Koncorde